_internals/configuration.md

```diff
@@ -118,329 +118,329 @@ When `ANSI_DEFAULTS` is enabled, it enables the following ISO settings:
 ODBC and OLE DB automatically set `ANSI_DEFAULTS` to `ON` when connecting.
 The driver and provider then set `CURSOR_CLOSE_ON_COMMIT` and
 `IMPLICIT_TRANSACTIONS` to `OFF`. So `CURSOR_CLOSE_ON_COMMIT` is actually `OFF` at
-connect time, but `ANSI_DEFAULTS` is `ON`. 
+connect time, but `ANSI_DEFAULTS` is `ON`.
 
-### babelfishpg_tsql.ansi_null_dflt_off
+### `babelfishpg_tsql.ansi_null_dflt_off`
 
-Modifies the behavior of the session to override the default nullability of new 
+Modifies the behavior of the session to override the default nullability of new
 columns when the ANSI `NULL` default option for the database is on.
 
-`ON` setting is not allowed for option `ANSI_NULL_DFLT_OFF`. 
+The setting `ON` is not allowed for option `ANSI_NULL_DFLT_OFF`.
 Please use `babelfishpg_tsql.escape_hatch_session_settings` to ignore this
 variable.
```

Comment:
Remove the message "Please use.... " that is a run-time message. Also, the text 'to ignore this variable' or 'to ignore errors' (as below) is not accurate.  Either remove all of these, or add a generic comment that there is an escape hatch and point to the escape hatch section.

Reply:
Removed.